MabiPale2/FrmMain.cs

```diff
@@ -164,7 +164,7 @@ private void BtnOpen_Click(object sender, EventArgs e)
 		private DialogResult ClearListQuestion()
 		{
 			if (LstPackets.Items.Count == 0)
-				return DialogResult.No;
+				return DialogResult.Yes;
```

Comment:
If clearing a blank list, the answer to "was the list cleared" can both be yes or no. For the purposes of deciding whether to update the current file name, the 'no' was changed to 'yes'.